lib/modules/manager/pixi/readme.md

```diff
@@ -1,5 +1,10 @@
-Limited support for pixi package manager
+### Pixi package manager
 
 Whenever the pixi config in `pyproject.toml` or `pixi.toml` file is updated, the `pixi.lock` file will be checked for updates as well.
 
-Currently only lock file maintenance is supported.
+Currently support channels from anaconda and prefix.dev, other channels is not suppoted yet.
+
+Renovatebot will pick pixi based on known lock file version and pixi version mapping,
+optionally you can set `constraints.pixi` in your config to override this.
+
+The minimal support version of pixi is `0.40.0`, when you are setting `constraints.pixi` in your config, please be carefully.
```

Comment:
Would be good to maybe mention why users need to be careful in that case :)

Reply:
this should be clear enough? if not would you mind help rephrase it?